pkg/vulnsrc/ubuntu/ubuntu.go

```diff
@@ -23,7 +23,7 @@ const (
 )
 
 var (
-	targetStatuses        = []string{"needed", "deferred", "released"}
+	targetStatuses        = []string{"ignored", "needed", "pending", "deferred", "released"}
```

Comment:
I checked all posible statuses:
`[not-affected DNE released ignored needs-triage needed deferred pending]`

we need to add `needs-triage` (as `Unknown` status (see https://github.com/aquasecurity/trivy-db/blob/11edc670d14bcb450e2f02738a54eaae8a2744bd/pkg/vulnsrc/ubuntu/ubuntu.go#L231-L232))

But i think we can just skip `not-affected` and `DNE` statuses to avoid skipping new statuses (if ubuntu will add them)
@knqyf263 wdyt?

Reply:
Status `needs-triage` has been added into targetStatuses.

Reply:
@DmitriyLewen `needs-triage` is "Under investigation". Trivy DB doesn't save such advisories.

Reply:
oh... right. comment about `needs-triage` confused me.
@skahn007gl can you exclude `needs-triage` from `targetStatuses` and remove [this comment](https://github.com/aquasecurity/trivy-db/blob/f6a5b03dc85c2147eb8195c8945177cd161ea2c5/pkg/vulnsrc/ubuntu/ubuntu.go#L231) to avoid confusing.

Reply:
@DmitriyLewen Done, test updated to.

Reply:
Thanks! fix linter error please